webpack.config.js

```diff
@@ -40,9 +38,5 @@ module.exports = {
       },
     ],
   },
-  //   output: {
-  //     filename: 'main.js',
-  //     path: path.resolve(__dirname, 'dist'),
-  //   },
```

Comment:
Great job! However, it would be best if you added a public path here in order for your CSS and js to be well rendered in the dist index.html
``publicPath: './'``